dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -157,7 +157,7 @@ def show
     @level = select_level
     return if redirect_under_13_without_tos_teacher(@level)
 
-    @body_classes = @script_level.script.background
+    @body_classes = @level.properties['background'] || @script_level.script.background
```

Comment:
should we remove the `background` property from script.rb with this change? do you know if any other scripts use it?

Reply:
Querying my local database, only `oceans` uses the background property

Reply:
I was thinking other scripts might still want this feature, but given that none had, I've now removed the support.